bismuth.spec

```
@@ -29,19 +30,25 @@ them via keyboard, just like in classical tiling window managers
 
 %files
 %license LICENSES/MIT.txt
+%{_libdir}/qt5/qml/org/kde/bismuth/core/qmldir
+%{_libdir}/qt5/qml/org/kde/bismuth/core/libbismuth_core.so
 %{_libdir}/qt5/plugins/kcms/kcm_bismuth.so
+%{_libdir}/qt5/plugins/org.kde.kdecoration2/bismuth_kdecoration.so
 %{_datadir}/config.kcfg/bismuth_config.kcfg
-%{_datadir}/icons/hicolor/32x32/status/bismuth*
-%{_datadir}/icons/hicolor/22x22/categories/bismuth-kcm.svg
-%{_datadir}/icons/hicolor/64x64/categories/bismuth-kcm.svg
-%{_datadir}/icons/hicolor/scalable/apps/bismuth.svg
-%{_datadir}/kconf_update/old_conf_ui.sh
-%{_datadir}/kconf_update/old_conf_ui.upd
-%{_datadir}/kpackage/kcms/kcm_bismuth
+%{_datadir}/kconf_update/bismuth*
+%{_datadir}/qlogging-categories5/bismuth.categories
 %{_datadir}/kservices5/kcm_bismuth.desktop
-%{_datadir}/kwin/scripts/bismuth
+%{_datadir}/kpackage/kcms/kcm_bismuth/*
```

No need to have the glob here since we will include everything
```suggestion
%{_datadir}/kpackage/kcms/kcm_bismuth
```